bokehjs/src/coffee/models/tools/actions/reset_tool.ts

```diff
@@ -0,0 +1,30 @@
+import {ActionTool, ActionToolView} from "./action_tool"
+import {Reset} from "core/bokeh_events"
+import * as p from "core/properties"
+
+export class ResetToolView extends ActionToolView {
+
+  model: ResetTool
+
+  doit(): void {
+    this.plot_view.clear_state()
+    this.plot_view.reset_range()
+    this.plot_view.reset_selection()
+    this.plot_model.plot.trigger_event(new Reset())
+  }
+}
+
+export class ResetTool extends ActionTool {
+  reset_size: boolean
```

Comment:
as an aside this could probably go away, since the resize tool went away

Reply:
I have a feeling that I removed this before, but maybe it was something else related to `ReiszeTool`. Anyway I started #7354 for this.